sciplot/specs/gridspecs.hpp

```diff
@@ -30,68 +30,77 @@
 #include <sciplot/specs/gridspecsbase.hpp>
 #include <sciplot/util.hpp>
 
-namespace sciplot {
-namespace internal {
+namespace sciplot
+{
 
 /// The class used to specify options for grid.
 class gridspecs : public gridspecsbase
 {
-public:
+  public:
     /// Construct a default gridspecs instance.
     gridspecs();
 
     /// Convert this gridspecs object into a gnuplot formatted string.
     auto repr() const -> std::string;
 
     /// Return a grid specification object for configuring grid lines along major xtics.
-    auto xtics() -> gridspecsbase& { return _gridmajor("xtics"); }
+    auto xtics() -> gridspecsbase & { return _gridmajor("xtics"); }
 
     /// Return a grid specification object for configuring grid lines along major ytics.
-    auto ytics() -> gridspecsbase& { return _gridmajor("ytics"); }
+    auto ytics() -> gridspecsbase & { return _gridmajor("ytics"); }
 
     /// Return a grid specification object for configuring grid lines along major ztics.
-    auto ztics() -> gridspecsbase& { return _gridmajor("ztics"); }
+    auto ztics() -> gridspecsbase & { return _gridmajor("ztics"); }
 
     /// Return a grid specification object for configuring grid lines along major rtics.
-    auto rtics() -> gridspecsbase& { return _gridmajor("rtics"); }
+    auto rtics() -> gridspecsbase & { return _gridmajor("rtics"); }
 
     /// Return a grid specification object for configuring grid lines along major x2tics.
-    auto x2tics() -> gridspecsbase& { return _gridmajor("x2tics"); }
+    auto x2tics() -> gridspecsbase & { return _gridmajor("x2tics"); }
 
     /// Return a grid specification object for configuring grid lines along major y2tics.
-    auto y2tics() -> gridspecsbase& { return _gridmajor("y2tics"); }
+    auto y2tics() -> gridspecsbase & { return _gridmajor("y2tics"); }
 
     /// Return a grid specification object for configuring grid lines along minor xtics.
-    auto mxtics() -> gridspecsbase& { return _gridminor("mxtics"); }
+    auto mxtics() -> gridspecsbase & { return _gridminor("mxtics"); }
 
     /// Return a grid specification object for configuring grid lines along minor ytics.
-    auto mytics() -> gridspecsbase& { return _gridminor("mytics"); }
+    auto mytics() -> gridspecsbase & { return _gridminor("mytics"); }
```

Comment:
I think it is more common to have `&` attached to the type (i.e. `gridspecsbase&`).

Reply:
This is somewhat of a religious thing. [Googles convention](https://google.github.io/styleguide/cppguide.html#Pointer_and_Reference_Expressions) is "use one convention per file and stick with it". Stackoverflow answers very. I don't care much, so I just set "PointerAlignment: Left". Will refomat the files.